scripts/logs-backup-from-all-pods/extract_logs.py

```diff
@@ -0,0 +1,187 @@
+import datetime
+import logging
+import subprocess
+import sys
+import json
+
+import psycopg2
+import redis
+import requests
+from datetime import datetime, timedelta
+
+def login_to_isd():
+    try:
+        cookie = ""
+        cmd = "curl -vvv -X POST '" + isd_gate_url + "/login?username=" + isd_admin_username + "&password=" + isd_admin_password + "&submit=Login'"
```

Comment:
Do we have any plans on what should be done when SAML is enabled ?

Reply:
sapor-gate will be used